editor.planx.uk/src/@planx/components/FindProperty/model.ts

```diff
@@ -48,15 +48,19 @@ export interface MinimumSiteAddress {
 
 // Full SiteAddress reflects selecting a record from the OS Places API "LPI" datasource
 export interface SiteAddress extends MinimumSiteAddress {
-  uprn?: string;
-  usrn?: string;
-  blpu_code?: string;
+  uprn: string;
+  usrn: string;
+  blpu_code: string;
   organisation?: string | null;
   sao?: string | null;
+  saoEnd?: string | null;
   pao?: string;
+  paoEnd?: string;
   street?: string;
   town?: string;
   postcode?: string;
+  ward?: string;
+  parish?: string;
```

Comment:
Generally a lot more "optional" types in here than I'd like - but OS Docs are really hard to make sense of about what's required versus not sometimes - for example, `PARISH_CODE` has a "multiplicity" of 1 (not 0..1) which I understand to mean required yet it's not one of the returned fields in the example response linked above.

Reply:
Helpful thanks!